.circleci/config.yml

```diff
@@ -270,23 +270,20 @@ workflows:
             - hmpps-community-accommodation-tier-2-ui-preprod
           requires:
             - request-preprod-approval
-      # - request-prod-approval:
-      #      type: approval
-      #      requires:
-      #        - deploy_preprod
-      # - hmpps/deploy_env:
-      #      name: deploy_prod
-      #      env: "prod"
-      #      jira_update: true
-      #      jira_env_type: production
-      #      slack_notification: true
-      #      slack_channel_name: << pipeline.parameters.releases-slack-channel >>
-      #      context:
-      #        - hmpps-common-vars
-      #        - hmpps-community-accommodation-tier-2-ui-prod
-      #      requires:
-      #        - request-prod-approval
-      #      helm_timeout: 5m
+      - request-prod-approval:
+           type: approval
+           requires:
+             - deploy_preprod
+      - hmpps/deploy_env:
+           name: deploy_prod
+           env: "prod"
+           slack_notification: true
+           slack_channel_name: << pipeline.parameters.releases-slack-channel >>
```

Comment:
is this parameter something that needs to be set on circle ci? https://app.circleci.com/settings/project/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fministryofjustice%2Fhmpps-community-accommodation-tier-2-ui